docs/events/hackathon.mdx

```diff
@@ -68,5 +68,11 @@ Leave a comment on this [GitHub issue](https://github.com/felvin-search/instant-
 
 - [Overview of Getting Started with Instant Apps](../instant-apps/getting-started.md)
 - [GitHub README and contributing guide](https://github.com/felvin-search/instant-apps)
+ 
+### Do the contributions I make during hacktoberfest count?
+- No. Only those made during the hackathon period (15-17) will count.
+
+### Will submissions on existing issues count?
+- Yes. But make sure you say you are working on an issue to avoid submissions of multiple pull requests.
```

Comment:
```suggestion
- Yes - you can work on existing issues. Do make sure to comment and say you are working on an issue to avoid submission of multiple pull requests. However, we do encourage thinking of new Instant Apps which could benefit your search experience. :)
```